package.json

```diff
@@ -14,7 +14,7 @@
     "publish": "npx lerna publish",
     "prelint": "npm run build",
     "pretest": "npm run build",
-    "test": "vitest --coverage"
+    "test": "vitest --coverage && npx lerna run test:jest --stream"
```

Comment:
I hate this but these two libraries need to be run through Jest to ensure that that works even though Vitest actually uses Jest's `expect` library under the hood.